apps/koala-frontend/src/app/app.component.ts

```diff
@@ -17,8 +19,9 @@ export class AppComponent implements OnInit {
 
   ngOnInit() {
     this.primengConfig.ripple = true;
-    this.translateService.setDefaultLang(LANGUAGE_CODE.GERMAN);
-    this.translateService.use(LANGUAGE_CODE.GERMAN); // TODO: use browser language
+    this.translateService.setDefaultLang(DEFAULT_LANGUAGE);
+    console.log('this.getBrowserLanguage()', this.getBrowserLanguage());
```

Comment:
Seems like forgotten to be removed :)

Reply:
removed it while you did the review 👍  thanks